setup.cfg

```diff
@@ -41,8 +41,7 @@ install_requires =
     jupyter_server>=1.16.0,<2
     notebook_shim>=0.1
     jinja2>=3.0.3
-    ypy-websocket>=0.1.6
-    jupyter_ydoc>=0.1.8
+    jupyter_ydoc==0.1.10
```

Comment:
A good practice is to explicit all directly used dependencies in Python package. As we are using ypy-websocket directly, it should still be mentioned here; although we may drop the version constrain if nothing specific is required for its usage withing JupyterLab.

Reply:
Thanks, indeed we can let `jupyter_ydoc` do the version constraint.